quint/src/flattening.ts

```diff
@@ -241,6 +241,10 @@ class Flatenner {
     const qualifiedName = def.defName ? undefined : def.qualifiedName ?? def.protoName
 
     const protoModule = this.importedModules.get(def.protoName)!
+    if (!protoModule) {
```

Comment:
When could you enter the `if` here?
You already have a non-null assertion on `get`?

```suggestion
    const protoModule = this.importedModules.get(def.protoName)!
    if (!protoModule) {
```

Reply:
oh, that's a good catch! I actually wanted to remove the `!` operator, as it makes the code crash with an unreadable stack trace.